GANDLF/compute/forward_pass.py

```diff
@@ -116,7 +118,7 @@ def validate_network(
         tqdm(valid_dataloader, desc="Looping over " + mode + " data")
     ):
         if params["verbose"]:
-            print("== Current subject:", subject["subject_id"], flush=True)
+            logger.debug("== Current subject:", subject["subject_id"])
```

Comment:
Does it work? AFAIR logger does not support handy string concat as print does and requires you (developer) to format log message manually beforehand like 
```suggestion
            logger.debug('== Current subject: {subject["subject_id"]}')
```